docs-en/chapter_array_and_linkedlist/linked_list.md

```diff
@@ -1,20 +1,20 @@
 # Linked Lists
 
-Memory space is a common resource for all programs. In a complex system environment, free memory space can be scattered throughout memory. We know that the memory space for storing an array must be contiguous, and when the array is very large, it may not be possible to provide such a large contiguous space. This is where the flexibility advantage of linked lists becomes apparent.
+Memory space is a shared resource among all programs. In a complex system environment, available memory can be dispersed throughout the memory. We understand that the memory allocated for an array must be continuous. However, for very large arrays, finding a sufficiently large contiguous memory space might be challenging. This is when the flexible advantage of linked lists becomes evident.
 
-A "linked list" is a linear data structure where each element is a node object, and the nodes are connected via "references". A reference records the memory address of the next node, allowing access to the next node from the current one.
+A "linked list" is a linear data structure in which each element is a node object, and the nodes are interconnected through "references". These references hold the memory addresses of subsequent nodes, enabling navigation from one node to the next.
 
-The design of a linked list allows its nodes to be scattered throughout memory, with no need for contiguous memory addresses.
+The design of linked lists allows for their nodes to be distributed across various memory locations without requiring contiguous memory addresses.
 
 ![Linked List Definition and Storage Method](linked_list.assets/linkedlist_definition.png)
 
-Observing the image above, the fundamental unit of a linked list is the "node" object. Each node contains two pieces of data: the "value" of the node and the "reference" to the next node.
+From the illustration in the figure above, we see that the basic building block of a linked list is the "node" object. Each node comprises two key components: the node's "value" and a "reference" to the next node.
 
-- The first node of a linked list is known as the "head node", and the last one is called the "tail node".
-- The tail node points to "null", which is represented as `null` in Java, `nullptr` in C++, and `None` in Python.
-- In languages that support pointers, like C, C++, Go, and Rust, the aforementioned "reference" should be replaced with a "pointer".
+- The initial node in a linked list is the "head node", while the final one is the "tail node".
```

Comment:
"initial" -> "first".
"while the final one" -> "and the last node" (while is only used for comparison or contrast).